CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Added `map<k,v>` support to the Worker Inspector window. [#1403](https://github.com/spatialos/gdk-for-unity/pull/1403)
 - Added `Open inspector V2` menu item that opens the new inspector in the browser. [#1407](https://github.com/spatialos/gdk-for-unity/pull/1407)
+- Moved Gdk Tools Configuration to the Unity "Project Settings" window. [#1408](https://github.com/spatialos/gdk-for-unity/pull/1408)
```

Comment:
This should go under a `### Changed` header rather than `Added` :) 

Feel free to add another heading under this one (you can see examples in the previous releases)